### DISCUSSION TITLE
iOS 16 incompatibility

### DISCUSSION THREAD

**Discussion by sandrods** [josefarias/hotwire_combobox#199] [Q&A]
Is that real a thing? I'm trying to test the examples in hotwirecombobox.com and they don't work either. 

Anybody had the same experience? Any workaround?

I REALLY don't want to remove it from our app.

**Comment by josefarias** [comment]
> Is that real a thing?

It might’ve been! When iOS 16 was released, the Safari version that came bundled with it had some quirks. They seem to have figured it out though. Seems like the combobox is working with latest Safari on mobile. But please let me know if not. Closing for now.
